metadata/src/test/java/org/apache/kafka/controller/PartitionChangeBuilderTest.java

```diff
@@ -411,7 +420,7 @@ public void testUncleanLeaderElection() {
                 .setIsr(Arrays.asList(2))
                 .setLeader(2)
                 .setLeaderRecoveryState(LeaderRecoveryState.RECOVERING.value()),
-            PARTITION_CHANGE_RECORD.highestSupportedVersion()
+            (short) 0 // TODO test against highest supported version
```

Comment:
```suggestion
                (short) 0 // TODO test against highest supported version
```

Reply:
Checkstyle doesn't complain about either variation. However, I find your suggestion slightly harder to read, it makes less obvious that `(short) 0` is a second argument to the ApiMessageAndVersion constructor, rather it look like it is part of the record declaration. What's your thinking behind the change?

Reply:
I don't mind either way as long as the other (short) 0s are aligned the same way. The reason I proposed this change is that the other (short) 0s were aligned like this.

Reply:
Hmm that is a good reason. I'll change it.